_studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp

```diff
@@ -6273,7 +6273,7 @@ mfxStatus ConfigureExecuteParams(
     }
 
 
-    if (true == executeParams.bSceneDetectionEnable && (FRC_ENABLED | config.m_extConfig.mode))
+    if (true == executeParams.bSceneDetectionEnable && (FRC_ENABLED & config.m_extConfig.mode))
```

Comment:
Logic is changed here. Is it done on purpose ?

Reply:
```cpp
(FRC_ENABLED | config.m_extConfig.mode)
```
This condition is always true. I thought '&' is meant to be here, because FrcMode enum contains bitmasks.

Reply:
also we can remove ```true ==``` as it don't have any sense as for me

Reply:
Done